RELEASE-NOTES.md

```diff
@@ -1,3 +1,32 @@
+### 0.11.0
+#### Compatibility
+- golem-messages: 2.24.0
+- golem-smart-contracts-interface: 1.6.1
+
+#### Endpoints and uses cases in general
+- Bugfix: Change concent response from ServiceRefused to Http404 (#1051).
```

Comment:
This is vague and uninformative. We did not replace all the `ServiceRefused` with HTTP 404 everywhere in Concent, did we?

Reply:
Nope, only when client want to communicate with "unsupported_protocol_version". I'll add that.